### DISCUSSION TITLE
Idea to put a Register in Player

### DISCUSSION THREAD

**Discussion by viniotero** [mz-ir/mazyar#117] [Ideas]
I know that is impossible for the past, but could create a register in the player from now, to show a ticket like OR or CR, showing that the guy bought the player with Closed Report (CR) or open report (OR). This would show some interested players that was bought in these two ways.

**Comment by mz-ir** [comment]
Mazyar operates locally on your browser and does not collect or aggregate data on a third-party server. Therefore, it cannot track every transfer happening every minute. Perhaps [Stats Xente Script](https://greasyfork.org/en/scripts/491442-stats-xente-script) could provide this functionality.
